commit 2a5b4f7a715921a232f67f6810268c6cd6aa0af2
Author: Florian Weimer <fweimer@redhat.com>
Date:   Fri Jul 8 12:08:48 2022 +0200

    elf: Rename tst-audit26 to tst-audit28
    
    tst-audit26 and tst-audit27 are already used by aarch64.
    
    Reviewed-by: Szabolcs Nagy <szabolcs.nagy@arm.com>

Conflicts:
	elf/Makefile
	  (Usual test backport differences.)

diff --git a/elf/Makefile b/elf/Makefile
index 3fae27d32676caf9..9e721d5d4e0a1cd9 100644
--- a/elf/Makefile
+++ b/elf/Makefile
@@ -353,7 +353,7 @@ tests += \
   tst-audit24d \
   tst-audit25a \
   tst-audit25b \
-  tst-audit26 \
+  tst-audit28 \
   tst-auditmany \
   tst-auxobj \
   tst-auxobj-dlopen \
@@ -658,7 +658,7 @@ modules-names = \
   tst-auditmod24c \
   tst-auditmod24d \
   tst-auditmod25 \
-  tst-auditmod26 \
+  tst-auditmod28 \
   tst-big-note-lib \
   tst-deep1mod1 \
   tst-deep1mod2 \
@@ -2145,10 +2145,10 @@ $(objpfx)tst-audit25b: $(objpfx)tst-audit25mod1.so \
 LDFLAGS-tst-audit25b = -Wl,-z,now
 tst-audit25b-ARGS = -- $(host-test-program-cmd)
 
-$(objpfx)tst-audit26: $(libdl)
-$(objpfx)tst-audit26.out: $(objpfx)tst-auditmod26.so
-$(objpfx)tst-auditmod26.so: $(libsupport) $(libdl)
-tst-audit26-ENV = LD_AUDIT=$(objpfx)tst-auditmod26.so
+$(objpfx)tst-audit28: $(libdl)
+$(objpfx)tst-audit28.out: $(objpfx)tst-auditmod28.so
+$(objpfx)tst-auditmod28.so: $(libsupport) $(libdl)
+tst-audit28-ENV = LD_AUDIT=$(objpfx)tst-auditmod28.so
 
 # tst-sonamemove links against an older implementation of the library.
 LDFLAGS-tst-sonamemove-linkmod1.so = \
diff --git a/elf/tst-audit26.c b/elf/tst-audit28.c
similarity index 100%
rename from elf/tst-audit26.c
rename to elf/tst-audit28.c
diff --git a/elf/tst-auditmod26.c b/elf/tst-auditmod28.c
similarity index 100%
rename from elf/tst-auditmod26.c
rename to elf/tst-auditmod28.c
